app/controllers/api/v1/users/omniauth_callbacks_controller.rb

```diff
@@ -23,12 +18,17 @@ def auth
           @auth ||= request.env['omniauth.auth']
         end
 
-        def render_success
-          render json: { success: true }, status: :ok
-        end
+        def render_response(user)
+          oauth_token = OauthToken.generate_access_token(user)
+          data = OauthTokenSerializer.new(oauth_token).serializable_hash[:data]
```

Comment:
Should we explicit this: `token_data`?
```suggestion
          token_data = OauthTokenSerializer.new(oauth_token).serializable_hash[:data]
```

Reply:
`data` is never a good variable 🙃